Guides/UPI/baremetal/nodes-custom-ip.md

```diff
@@ -0,0 +1,273 @@
+Setting Custom Node IPs
+---
+
+When deploying in non-standard network environments you may need to control what IPs are used by OKD and K8s services to ensure traffic flows over the correct networks/interfaces.
+
+Kubelet and other services used in the bootstrap don't use DNS to resolve IP addresses. An IP is either provided to the Kubelet command or one will be automatically discovered.
+
+By default OKD and K8s services often attempt to auto discover the IP to use based on the routability of each address. In a situation where you have multiple interfaces with valid routes, then the first valid IP will be used by interface order. This means in a system setup where you have `eno1` (routed public network) and `eno2` (routed private network), then the address on the `eno1` network will be used.
+
+> You can look at the runtime resolver for Node IPs [here](https://github.com/openshift/baremetal-runtimecfg/blob/master/cmd/runtimecfg/node-ip.go)
```

Comment:
This resolver is used in IPI case - is it being run in UPI too?

Reply:
I think so

Reply:
Yes, I'm on bare metal UPI and it's being ran